<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>255</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>225</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>465</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(244,244,244);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="9,0,0">
   <item row="2" column="0">
    <widget class="QPushButton" name="btnLogin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
    font-family:Microsoft Yahei;
    font:17px;
    color:white;
    background-color:rgb(14 , 150 , 254);
    border-radius:15px;
	font-weight:500;
    height:40px;
	padding-top:5px;
	padding-bottom:5px;
}

QPushButton:hover
{
    background-color:rgb(44 , 137 , 255);
}

QPushButton:pressed
{
    background-color:rgb(14 , 135 , 228);
}</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditPassword">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Yahei</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1.5px solid rgb(141,141,141);
font-family:Microsoft Yahei;
font:17px;
font-weight:600;
background-color:white;
padding-top:5px;
padding-bottom:5px;</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText</set>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPassword">
       <property name="styleSheet">
        <string notr="true">font-family:Microsoft Yahei;
font:17px;
color:#888888;</string>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelUsername">
       <property name="styleSheet">
        <string notr="true">font-family:Microsoft Yahei;
font:17px;
color:#888888;</string>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditUsername">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Yahei</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border:1.5px solid rgb(141,141,141);
font-family:Microsoft Yahei;
font:17px;
font-weight:600;
background-color:white;
padding-top:5px;
padding-bottom:5px;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
